aviator.js

```diff
@@ -363,7 +375,7 @@ Navigator.prototype = {
   getQueryString: function () {
     var uri, queryString;
 
-    if (this.pushStateEnabled) {
+    if (this.pushStateEnabled || !this.fallbackToHashRoutes) {
```

Comment:
`this.pushStateEnabled || this.fallbackToHashRoutes` could probably be refactored into a function since it's now used so much.